### PR TITLE
Replace deprecated `util.pump`

### LINKED ISSUE #142: Do not use deprecated `util.pump`, it's going away.
`util.pump` was deprecated [since nodejs v0.10](https://github.com/joyent/node/blob/v0.10.0-release/lib/util.js#L537-L538), and there is an ongoing discussion on removing that in the next major version: https://github.com/nodejs/node/pull/2531.

The line in question: [lib/cmd/common/local.js#L557](https://github.com/feedhenry/fh-fhc/blob/2.0.0+66/lib/cmd/common/local.js#L557):

``` javascript
util.pump(fileStream, response);
```

should be probably replaced with

``` javascript
fileStream.pipe(response);
```

_I am not making a pull request because I am not a user of this package and have not tested it._

### PR DESCRIPTION
**Pull request by deiga**
Fixes #142

### CODE REVIEW

**Comment by camilamacedo86** [comment]
@deiga . Thank you for your contribution. :1st_place_medal:  However, it is not building because it is not passing into the eslint rules as follows. Could you fix it and rebase the PR? Before doing the PR in order to test it you can run locally $grunt test to check if all is fine.   

```
/tmp/workspace/feedhenry_fh-fhc_PR-424-UZY3BGYSUF2XO7PELZ7SKJ3VUAL235UD75DTGQEAJULLZ5W4IOMA/lib/utils/exec.js
    7:5   error    'util' is defined but never used  no-unused-vars
   50:22  warning  Unexpected string concatenation   prefer-template
   53:13  warning  Unexpected string concatenation   prefer-template
   74:22  warning  Unexpected string concatenation   prefer-template
   75:26  warning  Unexpected string concatenation   prefer-template
   85:26  warning  Unexpected string concatenation   prefer-template
   89:29  error    Missing semicolon                 semi
  114:7   warning  Unexpected string concatenation   prefer-template
  126:18  warning  Unexpected string concatenation   prefer-template
  151:13  warning  Unexpected string concatenation   prefer-template

```

**Comment by deiga** [comment]
@camilamacedo86 I can't get `dtrace-provider` build on my macOS Sierra and node 8.1.4. Any advice? :)

**Comment by camilamacedo86** [comment]
@deiga could you check the readme of the project? In order to test it, you need just run locally $grunt test into the dir of the project.

File: package.json [hunk 1]
```diff
@@ -74,7 +74,7 @@
     "turbo-test-runner": "0.3.3"
   },
   "engines": {
-    "node": ">=4.4"
+    "node": "4.4.x"
```
Comment (camilamacedo86) [inline package.json]: The project works with versions upper than 4.4, so it should not be changed for now.

**Comment by camilamacedo86** [comment]
Hi @deiga, I am happy that you could fix it. Following what is required in order to approve this PR. 

1. Could you undo the change performed into the `package.json` file? ( Because of this project works with versions >= 4.4 and not just with this version and also it is important to highlight that your PR is related to `"Replace deprecated `util.pump`"` and it is not part of this scope)

2. Could you rebase it with the master branch?
Following the steps.         
```
$git checkout master
$git fetch upstream
$git pull upstream master
$git push
$git checkout <YOUR BRANCH>
$git rebase master
$git push origin +<YOUR BRANCH>
```
3. Could you bump the version?
You will update the files: npm-shrinkwrap.json, and package.json, sonar-project.properties with the upper version and create a commit for that as "Bump Version". Check the example: https://github.com/feedhenry/fh-fhc/pull/411/commits/c869ba286661304cb3dfaad09119f18bafd237f9

Really thank you for your contribution 👍

**Comment by deiga** [comment]
@camilamacedo86 I did your requested changes, but at least on my screen GitHub won't update the PR information :P

**Comment by camilamacedo86** [comment]
👍 LGTM, tested a command locally to check and see if all still working.

**Comment by camilamacedo86** [comment]
@deiga :1st_place_medal: thank you for your contribution.

**Pull request merged**